### ISSUE TITLE
RA_Firingtrap,Clusterbomb,Iceboundtrap missing level scaling and damage divide on targets

### ISSUE THREAD

**Issue by Paoly28** [rathena/rathena#1415]
`			md.damage = skill_lv * status_get_dex(src) + status_get_int(src) * 5 ;
			RE_LVL_TMDMOD();`

code doesn't have any scaling with base level hence why it only does 3~5k maximum damage. i don't even know if RE_LVL_TMDMOD(); does anything in this case

post-balancing formula is

ATK 100% + [{(Fire Trap Skill Level x Caster’s DEX) + (INT x 5)} x (1.5 + Caster’s Base Level / 100)] x {(Trap Research Skill Level x 20) / 100 }

so it's missing 3.25x modifier at level 175.

it also doesn't divide damage amongst target

**Comment by aleos89** [comment]
Did you happen to comment this out?
```
/// Renewal level modifier on damage
/// (disable by commenting the line)
///
// Leave this line to enable renewal base level modifier on skill damage (selected skills only)
#define RENEWAL_LVDMG
```

**Comment by Paoly28** [comment]
nope, RA_Firing/Icebound/Cluster code wasn't included in the structure, and re_lvl_tmdmod(); it does nothing

**Comment by Darkelfen** [comment]
RE_LVL_TMDMOD(); is translated as take this and multiply it by 1.5 + base level/100.

I've tested Firing Trap and Icebound Trap on my p-server (I haven't tested cluster bomb as we have a custom formula for it) on a ranger without weapon and all skills.
Stats Str, Dex and Int at 101 and job level at 1.

Tested at prontera dummies.

Damage at level 100 was 1336.
Damage at level 175 was 3817.

So, damage scaling by level is working.
